doc/libsc-build-wdeps.sh

```diff
@@ -33,15 +29,15 @@ bdie () {
 }
 
 # download, build and install zlib
-ZTAR="zlib-$ZVER.tar.gz"
-wget -N "https://www.zlib.net/$ZTAR"                    && \
-test `sha256sum "$ZTAR" | cut -d ' ' -f1` = "$ZSHA"     && \
+ZTAR="zlib.tar.gz"
+wget -N "https://www.zlib.net/current/zlib.tar.gz"      && \
+ZDIR=`tar -tzf "$ZTAR" | head -1 | cut -f1 -d"/"`       && \
```

Comment:
This line gives me `drwxr-xr-x madler`. What about using `perl -p -e 's/.*(zlib-[\d\.]+).*/\1/'`?

Reply:
The command `perl -p -e 's/.*(zlib-[\d\.]+).*/\1/'` does not terminate for me. I will have a closer look at it.

Reply:
> The command `perl -p -e 's/.*(zlib-[\d\.]+).*/\1/'` does not terminate for me. I will have a closer look at it.

In combination after the `head -1`, which is good?

Reply:
Ah, I misinterpreted the intended usage. Adding it to head -1 works, I have pushed an updated version.